.github/workflows/flatpak-test.yml

```diff
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+name: Flatpak
+jobs:
+  flatpak-builder:
+    name: Flatpak Builder
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: bilelmoussaoui/flatpak-github-actions@wip
+      with:
+        bundle: org.example.MyApp.Devel.flatpak
+        manifest: org.example.MyApp.yaml
```

Comment:
`manifest-path` instead of `manifest`, or the tests will always fail even with updated Docker image